src/tests/ut/doctor/doctor_test.cpp

```diff
@@ -1,17 +1,105 @@
 #include "doctor_test.hpp"
-
+#include "gender.hpp"
 namespace
 {
 
-TEST_F(DoctorTestFixture, test)
+class DoctorAppendVisitFunction : public DoctorTestFixture
+{
+};
+
+class DoctorRemoveVisitFunction : public DoctorTestFixture
+{
+};
+
+TEST_F(DoctorTestFixture, CreatingDoctorViaCreateShouldAppendDoctorsToClinic)
 {
     Doctor::createDoctor("Jan", "Kowalski", "80062378532", Gender::Male);
     Doctor::createDoctor("Janusz", "Schmit", "80062378532", Gender::Male);
     Doctor::createDoctor("Gawel", "Pawel", "80062378532", Gender::Male);
     Doctor::createDoctor("Jan", "Poniatowski", "80062378532", Gender::Male);
-    auto expected_doctor = Clinic::getDoctors().at(3);
 
-    EXPECT_EQ(Doctor::getDoctor("Jan", "Poniatowski").front(), expected_doctor);
+    auto first_expected_doctor = Clinic::getDoctors().at(0);
+    auto second_expected_doctor = Clinic::getDoctors().at(1);
+    auto third_expected_doctor = Clinic::getDoctors().at(2);
+    auto fourth_expected_doctor = Clinic::getDoctors().at(3);
+
+    EXPECT_EQ(Doctor::getDoctor("Jan", "Kowalski").front(), first_expected_doctor);
+    EXPECT_EQ(Doctor::getDoctor("Janusz", "Schmit").front(), second_expected_doctor);
+    EXPECT_EQ(Doctor::getDoctor("Gawel", "Pawel").front(), third_expected_doctor);
+    EXPECT_EQ(Doctor::getDoctor("Jan", "Poniatowski").front(), fourth_expected_doctor);
+}
+
+TEST_F(DoctorAppendVisitFunction, ShouldAppendGivenVisitToSpecifiedDoctor)
+{
+    Doctor::createDoctor("Jan", "Kowalski", "80062378532", Gender::Male);
+    Doctor::createDoctor("Jan", "Poniatowski", "80062378532", Gender::Male);
+
+    auto doctor_tested = Clinic::getDoctors().at(0);
+    auto irrelevant_doctor = Clinic::getDoctors().at(1);
+    createThreeSimpleVisits(irrelevant_doctor);
+    ASSERT_TRUE(doctor_tested->getVisits().empty());
+
+    const auto visit_to_append = irrelevant_doctor->getVisits().at(1);
+    doctor_tested->appendVisit(visit_to_append);
+    checkIfDoctorHasVisits(doctor_tested, {"second visit"});
+}
```

Comment:
Thinking of method Append use we should have a test in Visit which expects Visit to have doctor and doctor to have visit. 
It seems not necessary to have it here anyway. 
But as it's done let's keep it =)